Fix Warning for Empty Plugin File Name

HeyMehedi opened the pull request:
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

<!-- You can erase any parts of this template not applicable to your Pull Request. -->

### Changes proposed in this Pull Request:

<!-- If necessary, indicate if this PR is part of a bigger feature. Add a label with the format `focus: name of the feature [team:name of the team]`. -->

<!-- Describe the changes made to this Pull Request and the reason for such changes. -->

![image](https://github.com/user-attachments/assets/b8ed4c6a-0eb5-4dca-99ec-9648a586aa2c)


Closes # .

<!-- Begin testing instructions -->

### How to test the changes in this Pull Request:

<!-- Include detailed instructions on how these changes can be tested. Review and follow the guide for how to write high-quality testing instructions. -->

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Using PHP version < 8, deactivate all the Plugins
2. Activate the WooCommerce Only
3. Turn on the Debug 
4. You will see the Warning

<!-- End testing instructions -->

### Changelog entry

<!-- You can optionally choose to enter a changelog entry by checking the box below and supplying data. -->
<!-- It will trigger the 'Add changelog to PR' CI job to create and push the entry into the branch. -->

<!-- Due to org permissions, the job might fail for PRs crated from a fork under GitHub organizations. Possible solutions: -->
<!-- * Create entry manually with `pnpm --filter='@woocommerce/plugin-woocommerce' changelog add` and push it into the branch (replace `@woocommerce/plugin-woocommerce` with package name from nearest `package.json` file) -->
<!-- * Create entry from supplied PR data and push it automatically `pnpm utils changefile pr-number-here -o github-org-name-here` -->

-   [x] Automatically create a changelog entry from the details below.

<!-- If no changelog entry is required for this PR, you can specify that below and provide a comment explaining why. This cannot be used if you selected the option to automatically create a changelog entry above. -->

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

<!-- Choose only one -->

-   [ ] Patch
-   [x] Minor
-   [ ] Major

#### Type

<!-- Choose only one -->

-   [x] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message <!-- Add a changelog message here -->
Fix Warning for Empty Plugin File Name
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment <!-- If your Pull Request doesn't require a changelog entry, a comment explaining why is required instead -->

</details>

HeyMehedi committed:
update changelog

HeyMehedi committed:
Merge branch 'woocommerce:trunk' into trunk

HeyMehedi committed:
Update PluginUtil.php

HeyMehedi committed:
Merge branch 'trunk' into trunk

albarin commented:
Hi @HeyMehedi,
Thank you for your contribution! To help us with the review process, could you please fill in:
- The "Testing Steps" section
- The "Changelog Entry" section
in the PR description? These details will help ensure a smooth review process.

Thanks again for your work on this! 🙌

HeyMehedi committed:
Merge branch 'trunk' into trunk

HeyMehedi commented:
@albarin Thanks, I have updated the desc.

Commit:
Add changefile(s) from automation for the following project(s): woocommerce

albarin commented:
@HeyMehedi thanks for your update!
I've tweaked slightly the steps to include the PHP version and moved the changelog message to the correct section to be auto-generated. Somehow we ended up with 2 generated changelog messages, would you mind removing the file `plugins/woocommerce/changelog/2022-10-24-13-06-25-589622`? 🙏

HeyMehedi committed:
Merge branch 'trunk' into trunk

HeyMehedi committed:
Delete plugins/woocommerce/changelog/2022-10-24-13-06-25-589622

HeyMehedi commented:
> @HeyMehedi thanks for your update! I've tweaked slightly the steps to include the PHP version and moved the changelog message to the correct section to be auto-generated. Somehow we ended up with 2 generated changelog messages, would you mind removing the file `plugins/woocommerce/changelog/2022-10-24-13-06-25-589622`? 🙏

Thanks, I have removed the file.

albarin committed:
Merge branch 'trunk' into trunk
